Fakebook.Posts/Fakebook.Posts.DataAccess/Models/FakebookPostsContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace Fakebook.Posts.DataAccess.Models {
+    public partial class FakebookPostsContext : DbContext {
+        public FakebookPostsContext(DbContextOptions<FakebookPostsContext> options) : base(options) { }
+        public virtual DbSet<Post> Posts { get; set; }
+        public virtual DbSet<Comment> Comments { get; set; }
+        protected override void OnModelCreating(ModelBuilder modelBuilder) {
+            modelBuilder.Entity<Post>(entity => {
+                entity.ToTable("Post");
+                entity.Property(e => e.CreatedAt).HasDefaultValueSql("NOW()").ValueGeneratedOnAdd();
+                entity.Property(e => e.Picture).HasDefaultValue("").ValueGeneratedOnAddOrUpdate();
+            });
+            modelBuilder.Entity<Comment>(entity => {
+                entity.ToTable("Comment");
+                entity.Property(e => e.CreatedAt).HasDefaultValueSql("NOW()").ValueGeneratedOnAdd();
+                entity.HasOne(d => d.Post).WithMany(p => p.Comments).OnDelete(DeleteBehavior.Cascade);
+            });
+
+            OnModelCreatingPartial(modelBuilder);
+        }
+        partial void OnModelCreatingPartial(ModelBuilder builder);
```

Comment:
there's no point in adding this partial method, don't just copy it in without understanding what purpose it served in db-first.

Reply:
Removed. Not sure what it was there for, was copied from an older branch.